draft-irtf-cfrg-vdaf.md

```diff
@@ -2657,25 +2657,21 @@ def Sum(inp: Vec[Field128], joint_rand: Vec[Field128]):
 
 ### Prio3Histogram
 
-This instance of Prio3 allows for estimating the distribution of the
-measurements by computing a simple histogram. Each measurement is an arbitrary
-integer and the aggregate result counts the number of measurements that fall in
-a set of fixed buckets.
+This instance of Prio3 allows for estimating a distribution of measurements by
+computing a simple histogram. Each measurement increments one histogram bucket,
+out of a set of fixed buckets. The aggregate result counts the number of
```

Comment:
I think we need to be a bit more clear about what the measurements are. It may also be helpful to be a bit more concrete with an example.

```suggestion
computing a simple histogram. Each measurement is the index of the histogram bucket,
out of a fixed-length sequence of buckets. For example, the buckets might quantize the real numbers, and each measurement would report the bucket into which a client's value (a real number) would fall. The aggregate result counts the number of
```